.github/workflows/release-homebrew.yaml

```diff
@@ -0,0 +1,164 @@
+
+  # TODO: fix the arm64 test installer.
+  # Currently the first time of calling any Finch command in arm64 hosts triggered by Github action will fail by
+  # the error "Error: Process completed with exit code 137." So the arm64 job will fail.
+  # We temporarily use follow-up manual steps to complete and clean up it.
+  macos-arm64-test-installer:
+    needs: get-latest-tag
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          [
+            [self-hosted, macos, arm64, 11.7, release],
+            [self-hosted, macos, arm64, 12.6, release],
+            [self-hosted, macos, arm64, 13.0, release],
+          ]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    env:
+      ACCESS_TOKEN: ${{ secrets.FINCH_BOT_TOKEN }}
+      FINCH_TAG: ${{ needs.get-latest-tag.outputs.tag }}
+      FINCH_VESRION: ${{ needs.get-latest-tag.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.FINCH_TAG }}
+      - name: Clean up previous files
+        run: |
+          sudo rm -rf /opt/finch
+          sudo rm -rf ~/.finch
+          if pgrep '^qemu-system'; then
+            sudo pkill '^qemu-system'
+          fi
+          if pgrep '^socket_vmnet'; then
+            sudo pkill '^socket_vmnet'
+          fi
+          cd $(brew --repo homebrew/cask) 
+          brew uninstall --cask ./Casks/finch.rb || true
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Bump local cask version
+        run: |
+          brew update-reset 
+          cd $(brew --repo homebrew/cask)
+          echo "Update the local Finch cask with version ${FINCH_VERSION}"
+          brew bump-cask-pr --write-only -f --version=${FINCH_VERSION} finch
+      - name: Silently install
+        run: |
+          cd $(brew --repo homebrew/cask)
+          brew install --cask ./Casks/finch.rb
+      - name: Run e2e tests
+        run: INSTALLED=true make test-e2e
+      - name: Silently uninstall
+        if: ${{ always() }}
+        run: |
+          cd $(brew --repo homebrew/cask)
+          brew uninstall --cask ./Casks/finch.rb
+
+  macos-amd64-test-installer:
+    needs: get-latest-tag
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          [
+            [self-hosted, macos, amd64, 10.15, release],
+            [self-hosted, macos, amd64, 11.7, release],
+            [self-hosted, macos, amd64, 12.6, release],
+            [self-hosted, macos, amd64, 13.0, release],
+          ]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    env:
+      ACCESS_TOKEN: ${{ secrets.FINCH_BOT_TOKEN }}
+      FINCH_TAG: ${{ needs.get-latest-tag.outputs.tag }}
+      FINCH_VESRION: ${{ needs.get-latest-tag.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.FINCH_TAG }}
+      - name: Clean up previous files
+        run: |
+          sudo rm -rf /opt/finch
```

Comment:
I think postinstall script is doing the clean up, better skip the clean up explicitly which will be more similar to actual use case?

Reply:
Could you give more context about the postinstall script? What is it and how is it triggered?